libs/speculos-transport/src/index.ts

```diff
@@ -181,6 +201,12 @@ export async function createSpeculosDevice(
           `${dependency}:./apps/${conventionalAppSubpath(model, firmware, dependency, appVersion)}`,
         ]
       : []),
+    ...(dependencies !== undefined
+      ? dependencies.flatMap(dependency => [
+          "-l",
+          `${dependency.name}:./apps/${conventionalAppSubpath(model, firmware, dependency.name, dependency.appVersion ? dependency.appVersion : "1.1.0")}`,
```

Comment:
Why do we check if `dependency.appVersion` is defined in three different location ?

Reply:
needed a fallback value since `appVersion` is optional